driverkit/Makefile

```diff
@@ -5,12 +5,16 @@ DRIVERKIT ?= $(shell which driverkit)
 CONFIGS := $(wildcard config/*/*.yaml)
 VERSIONS := $(patsubst config/%,%,$(sort $(dir $(wildcard config/*/))))
 VERSIONS := $(VERSIONS:/=)
-TARGET_DISTRO ?=
+TARGET_DISTRO ?= *
+TARGET_KERNEL ?= *
 S3_DRIVERS_BUCKET ?= "falco-distribution/driver"
 
 all: $(patsubst config_%,%,$(subst /,_,$(CONFIGS)))
 
-specific_target: $(patsubst config_%,%,$(subst /,_,$(wildcard config/*/${TARGET_DISTRO}_*)))
+specific_target: $(patsubst config_%,%,$(subst /,_,$(wildcard config/*/${TARGET_DISTRO}_${TARGET_KERNEL}-*)))
```

Comment:
Just an idea here, Max.

What if we make this filter broader? I mean, something like `TARGET_KERNEL=5.x` would be very cool and needed as filter.

Consider we already have `make <driver_versiion>_<distro>_<kernel>.yaml` make commands generated.

Run this to list all the make commands available:

```console
make -qp | awk -F':' '/^[a-zA-Z0-9][^$#\/\t=]*:([^=]|$)/ {split($1,A,/ /);for(i in A)print A[i]}' | sort -u
```

Reply:
I like the idea! I'll try to do it

Reply:
It works Max!

Also added the same filtering feature for Falco driver versions.

Reply:
Thank you